### PR TITLE
[Feature] Replace moment with datefns

### PR DESCRIPTION
**Pull request by daiHash**
## Description
Replace moment with datefns

## Closes issue(s)
#80 

## Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

## Checklist

- [x] I have tested this code

### CODE REVIEW

File: src/pages/GardenView/index.tsx [hunk 1]
```diff
@@ -61,7 +62,7 @@ export const GardenView = () => {
       ruleId: rule._id || "",
       // TODO: Fix when backend updates schema for completedTask's fireBaseUserId
       fireBaseUserId: userData?.id || "",
-      date: moment.utc().startOf("day").toDate(),
+      date: new Date(),
```
Comment (kome12) [inline src/pages/GardenView/index.tsx:65]: @daiHash Is this creating a date in UTC? It's a lot easier to handle dates and times in UTC and then convert it to local time on the frontend if we need to show date / time in the frontend.

**Review by kome12 — COMMENTED** [review]
@daiHash I'm fine to change the package to date-fns. I just had 1 question about the date creation so if you can get back to me on that, that'll be great!

File: src/pages/GardenView/index.tsx (hunk 1 above)
Comment (conalli) [reply src/pages/GardenView/index.tsx:65]: I just checked the documentation, and it looks like this should be a UTC date. Would be good to test though.

Comment (kome12) [reply src/pages/GardenView/index.tsx:65]: Okay, as long as it's UTC, it should be fine!

**Review by kome12 — APPROVED** [review]
LGTM!

File: src/pages/GardenView/index.tsx (hunk 1 above)
Comment (daiHash) [reply src/pages/GardenView/index.tsx:65]: I 'll check it tonight🙇‍♂️

**Pull request merged**